bigquery/README.md

```diff
@@ -12,21 +12,24 @@ if you haven't done so already.
     bq --project_id MY_GOOGLE_CLOUD_PROJECT mk platinum_genomes
     bq --project_id MY_GOOGLE_CLOUD_PROJECT cp google.com:biggene:platinum_genomes.sample_info  MY_GOOGLE_CLOUD_PROJECT:platinum_genomes.sample_info
     ```
-  * Change project ids in `config/platinum_genomes/facet_fields.csv`.
+  * Change project ids in `dataset_config/platinum_genomes/facet_fields.csv`.
```

Comment:
This step should be no longer necessary right?

Reply:
In that case, was MY_GOOGLE_CLOUD_PROJECT unintentionally changed to google.com:api-project-360728701457 in this PR?

Reply:
Done. Looking forward to no longer having to change this file. :)